build_defs/build_defs.bzl

```diff
@@ -1,111 +1,14 @@
-load(":run_in_repo.bzl", "EXECUTION_REQUIREMENTS_TAGS", "run_in_repo", "run_in_repo_test")
-load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
-load("//build_defs/kotlin_native:build_defs.bzl", "kt_wasm_binary", "kt_wasm_library")
+"""Arcs BUILD rules."""
 
-_ANDROID_TARGETS = False
```

Comment:
It's true that there's currently no Android targets, but as soon as we introduce particle accelerator, there will be Android targets.

Reply:
They're going to be done in a very different way from how wasm particles are built and bundled. It's a separate beast entirely, and they're going to need fairly different build rules.

At some point there might be code shared between wasm and android particles, and that code could possibly benefit from a macro which generates both targets, but for now I don't think it's necessary.

Reply:
A production particle (particle accelerator jvm), and a dev particle (wasm) really shouldn't have any code differences at all.  The same particle, unchanged, should run in both contexts. 

Perhaps you're thinking of particles that can call native JVM functions directly?

Reply:
Actually now that I think about it, if kt_library() ultimately supports Kotlin Multiplatform, we'll get multi-target for free anyway.